parquet-column/src/main/java/org/apache/parquet/column/statistics/IntStatistics.java

```diff
@@ -19,11 +19,33 @@
 package org.apache.parquet.column.statistics;
 
 import org.apache.parquet.bytes.BytesUtils;
+import org.apache.parquet.column.statistics.bloomFilter.BloomFilter;
+import org.apache.parquet.column.statistics.bloomFilter.BloomFilterOpts;
+import org.apache.parquet.column.statistics.bloomFilter.BloomFilterStatistics;
 
-public class IntStatistics extends Statistics<Integer> {
+import org.apache.commons.lang.ArrayUtils;
 
+import java.util.Arrays;
+import java.util.List;
+
+public class IntStatistics extends Statistics<Integer> implements BloomFilterStatistics<Integer> {
   private int max;
   private int min;
+  private BloomFilter bloomFilter;
+  private boolean isBloomFilterEnabled;
+
+  public IntStatistics(){
+  }
```

Comment:
we should limit multiple constructors. If we don't need it anymore let's not add a parameterless constructor and just update all calls.